### DISCUSSION TITLE
Remove Login Page

### DISCUSSION THREAD

**Discussion by nox0x1** [prasathmani/tinyfilemanager#546] [Q&A] [answered]
First of all, thank you for creating and sharing this, it's amazing and I'll be sending Paypal donations your way to show my appreciation.
I was able to fully customize it to my application, but have one last remaining item.

I'm using v2.4.5, what portion of the file do I delete to remove the Login page so that the users are taken straight to the directory as user authentication is being handled by the site itself which this is a part of?

Thank you again.

**Comment by prasathmani** [comment, marked as answer]
just set `$use_auth = false;` to disable the login page
![image](https://user-images.githubusercontent.com/11345598/118384391-4b65b780-b623-11eb-93b3-54e329cdaeb4.png)

**Reply by nox0x1** [reply]
Thank you Sir, I somehow missed it but now working perfectly.

**Reply by 17500mph** [reply]
When did this get added? :)
Cool. Now I don't need to slide in a funny line... 
Though what I'm doing is using two versions - one that does not have the funny line so that I can still login as admin to 'do stuff'.

**Comment by martiniturbide** [comment]
Hello

Is there a way to remove the login page and maintain the default security as "user", with only permissions to view files. 
And have some other way to login with "admin" user to have access to edit, delete, etc.

Regards

**Reply by prasathmani** [reply]
```php
$use_auth = false;
$global_readonly = true;
```

**Reply by martiniturbide** [reply]
Thanks for the reply @prasathmani 
But if I set that, is there a way that an "admin" user can login and have access to change/delete file?

Regards

**Reply by prasathmani** [reply]
@martiniturbide, you can create an additional copy of TinyFileManager and activate login with admin rights, it allowing you to edit and remove all activities.
